src/test/java/ludogorie_soft/reservations_platform_api/service/impl/CalendarServiceImplTest.java

```diff
@@ -102,58 +96,85 @@ void testGetMyCalendarReturnsFileNameSuccessfully() throws IOException {
         when(propertyRepository.findById(propertyId)).thenReturn(Optional.of(property));
         when(bookingRepository.findByPropertyId(propertyId)).thenReturn(Collections.singletonList(booking));
 
+        //WHEN
         String result = calendarService.getMyCalendar(propertyId);
 
+        //THEN
         assertEquals(propertyId + ".ics", result);
         verify(propertyRepository, times(1)).save(property);
     }
 
     @Test
     void testGetMyCalendarShouldThrowWhenPropertyNotFound() {
+        //WHEN
         when(propertyRepository.findById(propertyId)).thenReturn(Optional.empty());
 
+        //THEN
         assertThrows(IllegalArgumentException.class, () -> calendarService.getMyCalendar(propertyId));
     }
 
     @Test
     void testSyncAirBnbCalendarSuccessfully() throws IOException, ParserException {
+        //GIVEN
         Property property = new Property();
         property.setId(propertyId);
         property.setAirBnbICalUrl("https://bg.airbnb.com/calendar/ical/1247739757393025285.ics?s=743aa2e810329f14445dd5f53a6279f8");
 
         when(propertyRepository.findById(propertyId)).thenReturn(Optional.of(property));
 
+        //WHEN
         calendarService.syncAirBnbCalendar(propertyId);
 
+        //THEN
         verify(propertyRepository, times(1)).findById(propertyId);
     }
 
     @Test
-    void testSyncForAvailableDatesReturnsTrue() {
+    void testSyncForAvailableDatesReturnsTrueWhenFileExists() throws ParserException, IOException {
+        //GIVEN
         String filePath = "test.ics";
         Date startDateRequest = new Date();
         Date endDateRequest = new Date();
 
-        File file = mock(File.class);
-        when(file.exists()).thenReturn(true);
-
+        //WHEN
         boolean result = calendarService.syncForAvailableDates(filePath, startDateRequest, endDateRequest);
 
+        //THEN
         assertTrue(result);
     }
 
     @Test
-    void testSyncForAvailableDatesReturnFalse() {
+    void testSyncForAvailableDatesReturnsTrueWhenNoEventsOverlap() throws ParserException, IOException {
+        //GIVEN
+        Date startDateRequest = new Date(System.currentTimeMillis() + 86400000 * 10);
```

Comment:
You can add a constant so it ore clear what this number stands for - for example `ONE_DAY_IN_MILLISECONDS=86400000`